pyproject.toml

```diff
@@ -110,10 +110,12 @@ line_length = 88
 minversion = "7.0"
 testpaths = "tests"
 xfail_strict = true
+markers = [
+    "slow: marks tests as slow (deselect with '-m \"not slow\"')",
+]
```

Comment:
I couldn't spot using this marker in `Makefile`. Is it intended ?

Reply:
Yes, it's used as a `PYTEST_ADDOPTS` item for the [slow tests](https://github.com/canonical/craft-platforms/actions/runs/12058519983/job/33625373752?pr=60#step:6:5)